runtime-modules/council/src/lib.rs

```diff
@@ -552,12 +557,35 @@ impl<T: Trait> ReferendumConnection<T> for Module<T> {
     }
 
     /// Check that it is a proper time to release stake.
-    fn can_release_vote_stake() -> Result<(), Error<T>> {
-        // ensure it's proper time to release stake
-        match Stage::<T>::get().stage {
-            CouncilStage::Idle => (),
-            _ => return Err(Error::CantReleaseStakeNow),
-        };
+    fn can_release_vote_stake(
+        vote: &CastVoteOf<T>,
+        current_voting_cycle_id: &u64,
+    ) -> Result<(), Error<T>> {
+        // allow release for very old votes
+        if vote.cycle_id > current_voting_cycle_id + 1 {
```

Comment:
This test is supposed to unlock very old votes, in which case the test seems wrong, because old votes would have a a much _lower_ cycle ID than the current one, this test checks the opposite?

Reply:
Yes, you are right. The two variables were mistakenly exchanged in this condition.